actors/common/src/main/java/org/sunbird/learner/util/Util.java

```diff
@@ -1608,6 +1652,30 @@ public static void addMaskEmailAndPhone(Map<String, Object> userMap) {
     return organisations;
   }
 
+  private static Map<String, Map<String, Object>> getEsResultByListOfIds(
```

Comment:
Move this function to elasticSearchUtil - this can be re-used for other occassions - and is completely generic as well.

local variable names need to be renamed while moving to make it generic